src/main.js

```diff
@@ -1,9 +1,13 @@
 import IntlPolyfill from "./core.js";
 
 // hack to export the polyfill as global Intl if needed
-if (typeof Intl !== undefined) {
-    Intl = IntlPolyfill;
-    IntlPolyfill.__applyLocaleSensitivePrototypes();
+if (typeof Intl !== 'undefined') {
+    try {
```

Comment:
I'm not sure about this try/catch, and the silence failure when trying to apply sensitive prototypes.